README.md

```diff
@@ -161,20 +161,25 @@ https://console.developers.google.com: click on "Select a project..."
 in the top navbar and then "Create a project."
 
 You will then need a name for your AppEngine project.  Let's suppose
-you call it `mycompany-snippets`.
+you call it `mycompany-snippets`. You must then set the project by 
+running `gcloud config set project mycompany-snippets` from the snippets
```

Comment:
+1 -- it also makes it harder to accidentally deploy this code to another app (which is something I've almost done with several apps).